rpc/contract_test.go

```diff
@@ -475,6 +478,8 @@ func TestEstimateMessageFee(t *testing.T) {
 }
 
 func TestEstimateFee(t *testing.T) {
+	//TODO: upgrade the mainnet and testnet test cases before merge
+
```

Comment:
Maybe we can just add another t.Skip("insert note here") instead of commenting most of the tests out

Reply:
I chose to keep the tests instead of skipping them because the mock environment is running without any problem; what do you think?

Reply:
I was thinking the same, but thought t.Skip() might have been useful when most the tests are commented out. It also helps keep track of which tests we have turned off, and need updating. Also long as we are consistent and leave a note, it should be fine.